### PR TITLE
DOC-4186: Added the support bundles content

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

However, I think we should create a tool/script as part of Corda package which allows a user to create the support bundle.
@davidcurrie / @ronanbrowne / @liamgR3 Thoughts ?

**Comment by yarivus** [comment]
> > However, I think we should create a tool/script as part of Corda package which allows a user to create the support bundle.
> 
> There's a post-it on a lucidchart to do this. One thought was to use https://troubleshoot.sh/ as this also allows us to do some other things like pre/post-install checks.

@davidcurrie is this post-it represented as a backlog work item ?
Reading the instructions - it would take us 10 mins to create such a script ... so I hope this work item is not coming so late ;)

**Comment by davidcurrie** [comment]
That reminds me though, one of the other features Troubleshoot has is redaction. We should, however, do an exercise before Alpha release to ensure that we're not leaking any of the credentials we're using into the logs.

**Comment by davidcurrie** [comment]
> @davidcurrie is this post-it represented as a backlog work item ?
Reading the instructions - it would take us 10 mins to create such a script ... so I hope this work item is not coming so late ;)
In the ruthless back prioritisation it was deemed to be a post-beta item and, as with all our post-beta work, Lucid is the only place it is represented. Feel free to take the 10 mins though! 😛

**Comment by yarivus** [comment]
> > @davidcurrie is this post-it represented as a backlog work item ?
> > Reading the instructions - it would take us 10 mins to create such a script ... so I hope this work item is not coming so late ;)
> 
> In the ruthless back prioritisation it was deemed to be a post-beta item and, as with all our post-beta work, Lucid is the only place it is represented. Feel free to take the 10 mins though! 😛

I should at least be in JIRA - so please create such a work item for post beta.

**Comment by davidcurrie** [comment]
> I should at least be in JIRA - so please create such a work item for post beta.

That's a discussion you should have with PM. At least for this team, there is a lot of post-beta content that only exists in Lucid as that's where Milan was doing the release planning.

**Comment by yarivus** [comment]
> > I should at least be in JIRA - so please create such a work item for post beta.
> 
> That's a discussion you should have with PM. At least for this team, there is a lot of post-beta content that only exists in Lucid as that's where Milan was doing the release planning.

@milan-khan - please acknowledge this work item will be included post beta

**Review by oscardalmau-r3 — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md [hunk 1]
```diff
@@ -9,3 +9,32 @@ menu:
...
 ---
 <!--https://r3-cev.atlassian.net/browse/CORE-7232-->
+
+In the event of an issue, you need to know what information R3 requires from you to be able to perform problem determination. This collection of information is referred to as a support bundle. The following document lists the steps to troubleshoot a Corda deployment in Kubernetes and collect the support material.
+
+## Set Kubernetes Context
+
+Set Kubernetes context to use the namespace where Corda is installed by default. Replace `$CORDA_NAMESPACE` in the following command by your namespace name:
+
+```
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md:19]:
```suggestion
```shell
```

File: content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md [hunk 2]
```diff
@@ -9,3 +9,32 @@ menu:
 section_menu: corda-5-alpha
 ---
 <!--https://r3-cev.atlassian.net/browse/CORE-7232-->
+
+In the event of an issue, you need to know what information R3 requires from you to be able to perform problem determination. This collection of information is referred to as a support bundle. The following document lists the steps to troubleshoot a Corda deployment in Kubernetes and collect the support material.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md:13]: Seems a bit wordy.. Maybe something like:
This page describes how to gather the information required by R3 to assist you with troubleshooting in the event of an issue with a Kubernetes deployment.

We probably need a general team consensus as to whether we use document/section/page..

File: content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md [hunk 3]
```diff
@@ -9,3 +9,32 @@ menu:
...
+
+## Set Kubernetes Context
+
+Set Kubernetes context to use the namespace where Corda is installed by default. Replace `$CORDA_NAMESPACE` in the following command by your namespace name:
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md:17]:
```suggestion
Set Kubernetes context to use the namespace where Corda is installed by default. Replace `$CORDA_NAMESPACE` in the following command with your namespace name:
```

File: content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md [hunk 4]
```diff
@@ -9,3 +9,32 @@ menu:
...
+
+```
+kubectl config set-context --current --namespace=$CORDA_NAMESPACE
+```
+
+## Collect Helm Material
+
+* Get Corda Helm chart release name, version and app version: `helm ls`
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md:25]:
```suggestion
* Get Corda Helm chart release name, version, and app version: `helm ls`
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md:13]: Agree. Updated the short desc.

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thanks @AnnaK-R3
